core/src/main/java/com/linecorp/armeria/server/Http1RequestDecoder.java

```diff
@@ -160,10 +161,13 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                     // immutability.
                     final boolean hasInvalidExpectHeader = !handle100Continue(id, nettyReq);
 
+                    final String path = HttpHeaderUtil
+                            .maybeTransformAbsoluteUri(nettyReq.uri(), cfg.absoluteUriTransformer());
```

Comment:
I wanted to make sure that we want to pass the original path as an input of `absoluteUriTransformer`, right?

Reply:
Right 😄   I think this was the original implementation as well.

https://github.com/line/armeria/blob/5f6a3b4836a9f0e15d4d074729054993597888c8/core/src/main/java/com/linecorp/armeria/internal/common/ArmeriaHttpUtil.java#L621